packages/database_v2/__WIP_pg_schema_review/artifacts/schema.graphql

```diff
@@ -0,0 +1,651 @@
+type Block {
```

Comment:
Can we leave this out for now, I think it out of scope here and just adds noise.